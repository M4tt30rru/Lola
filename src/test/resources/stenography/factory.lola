##Import "std.lola"
##Find
 ##Type(_type) ##Identifier(_name)(##ArgumentDeclarationList(_l)) -> produce;
 ##replace ##(_type) ##(_name) (##(_l)) {return new ##(_type)(##(",".join(a._name.name for a in _l._args)));}

T<String> f(int x) -> produce;
