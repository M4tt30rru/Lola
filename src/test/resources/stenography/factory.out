T<String> f(int x) {
 return new T<String>(x);
}
